plugins/gui/src/selection_details_widget/tree_navigation/selection_tree_proxy.cpp

```diff
@@ -40,8 +39,9 @@ namespace hal
         return comparison;
     }
 
-    void SelectionTreeProxyModel::applyFilterOnGraphics()
+    /*void SelectionTreeProxyModel::applyFilterOnGraphics()
     {
+        qDebug()<<"called applyFilterOnGraphics()";
```

Comment:
remove?

Reply:
It seems I missed that. Yes, we can remove that statement.